qemu/tests/cpu_flag_intel_pt.py

```diff
@@ -0,0 +1,64 @@
+import logging
+
+from avocado.utils import process
+from virttest import error_context, env_process
+from provider.cpu_utils import check_cpu_flags
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    Test cpu flag intel-pt.
+    1) Check if current flags are in the supported lists, if no, cancel test
+    2) Otherwise, set pt_model = 1 on host at first
```

Comment:
It's `pt_mode` not `pt_model`, pls update the rest as well.

Reply:
thanks and update